static/js/components/inputs/SelectField.js

```diff
@@ -135,25 +109,30 @@ export default class SelectField extends React.Component {
     }
   };
 
-  autocompleteProps: Function = (): Object => {
+  getAutocompleteProps: Function = (): Object => {
     const {
+      autocompleteBehaviors,
+      autocompleteStyleProps,
       label,
       options,
       keySet,
       errors
     } = this.props;
+    // this.props.autocompleteBehaviors is passed in as a single function or a list of functions
```

Comment:
Can we change this to only accept a list of functions? We only use it without the list in one place and it simplifies this check below